modules/sort/test/spec.coffee

```diff
@@ -108,3 +141,52 @@ describe 'hx-sort tests', ->
         if supportsOptions()
           array.sort(hx.sort.localeCompare('sv')).should.eql(['a', 'ä', 'e', 'é', 'è', 'z'])
           array.sort(hx.sort.localeCompare('de')).should.eql(['a', 'e', 'é', 'è', 'z', 'ä'])
+
+  describe 'using localeCompare potentially with Intl.Collator. ', localeCompareTests
+
+  describe 'using localeCompare without Intl.Collator. ', runWithoutCollator(localeCompareTests)
+
+  compareNullsLastTests = ->
+    it 'Strings should return the right value', ->
+      hx.sort.compareNullsLast('a', 'b').should.equal(-1)
+      hx.sort.compareNullsLast('b', 'a').should.equal(1)
+      hx.sort.compareNullsLast('a', 'a').should.equal(0)
+
+    it 'undefined should always be sorted to the end', ->
+      hx.sort.compareNullsLast(undefined, 'a').should.equal(1)
+      hx.sort.compareNullsLast('a', undefined).should.equal(-1)
+      hx.sort.compareNullsLast(undefined, 'z').should.equal(1)
```

Comment:
That change does make sense but would mean making every comparison with `hx.sort.compare()(a,b)` which breaks the current api - I don't mind doing it but I'm not sure how we'd do it 'safely - unless we had `hx.sort.compare(a,b,options)`?

Also, we could add the `nullsLast` option to the `localeCompare` function as it already has an options object which would at least solve that issue.